### DISCUSSION TITLE
Check-ins with TAGs a part of annual sandbox reviews

### DISCUSSION THREAD

**Discussion by joshgav** [cncf/toc#1208] [Ideas]
CNCF asks sandbox projects to submit an annual review of their technical and governance state. We recently asked TAGs to handle accepting those annual reviews, but then we realized much of the review of the review (sic) could be automated, and so we intend to handle future ones automatically.

I wanted to suggest that although the report will be automated, as part of the annual review let's ask projects to come to their TAGs for a 10-minute readout and Q&A on the report. This check-in would keep the project engaged with CNCF and related projects and ensure the maintainers and CNCF representatives get a chance at least once a year to talk over any concerns or opportunities that have arisen.

**Comment by TheFoxAtWork** [comment]
This may be largely contingent upon https://github.com/cncf/toc/discussions/1205
